docs/reference/release-notes/7.7.asciidoc

```diff
@@ -1,22 +1,612 @@
+
+Infra/Packaging::
+* Introduce aarch64 packaging {pull}53914[#53914]
+
+Machine Learning::
+* Implement ILM policy for .ml-state* indices {pull}52356[#52356] (issue: {issue}29938[#29938])
+
+Mapping::
+* Introduce a `constant_keyword` field. {pull}49713[#49713]
+
+SQL::
+* constant_keyword support {pull}53241[#53241] (issue: {issue}53016[#53016])
+* Add optimisations for not-equalities {pull}51088[#51088] (issue: {issue}49637[#49637])
+* Add support for passing query parameters in REST API calls {pull}51029[#51029] (issue: {issue}42916[#42916])
+
+Search::
+* Introduce compound query {pull}52387[#52387] (issue: {issue}51967[#51967])
+* Add a cluster setting to disallow expensive queries {pull}51385[#51385] (issue: {issue}29050[#29050])
+* Add new x-pack endpoints to track the progress of a search asynchronously {pull}49931[#49931] (issue: {issue}49091[#49091])
+
+
+
+[[enhancement-7.7.0]]
+[float]
+=== Enhancements
+
+Aggregations::
+* Try to save memory on aggregations {pull}53793[#53793]
+* Speed up partial reduce of terms aggregations {pull}53216[#53216] (issue: {issue}51857[#51857])
+* Simplify SiblingPipelineAggregator {pull}53144[#53144]
+* Add histogram field type support to boxplot aggs {pull}52265[#52265] (issues: {issue}33112[#33112], {issue}52233[#52233])
+* Percentiles aggregation validation checks for range {pull}51871[#51871] (issue: {issue}51808[#51808])
+* Begin moving date_histogram to offset rounding (take two) {pull}51271[#51271] (issues: {issue}50609[#50609], {issue}50873[#50873])
+* Password-protected Keystore Feature Branch PR {pull}51123[#51123] (issues: {issue}32691[#32691], {issue}49340[#49340])
+* Implement top_metrics agg {pull}51155[#51155] (issue: {issue}48069[#48069])
+* Bucket aggregation circuit breaker optimization. {pull}46751[#46751]
+
+Analysis::
+* Removes old Lucene's experimental flag from analyzer documentations {pull}53217[#53217]
+
+Authentication::
+* Add exception metadata for disabled features {pull}52811[#52811] (issues: {issue}47759[#47759], {issue}52311[#52311], {issue}55255[#55255])
+* Validate role templates before saving role mapping {pull}52636[#52636] (issue: {issue}48773[#48773])
+* Add support for secondary authentication {pull}52093[#52093]
+* Expose API key name to the ingest pipeline {pull}51305[#51305] (issues: {issue}46847[#46847], {issue}49106[#49106])
+* Disallow Password Change when authenticated by Token {pull}49694[#49694] (issue: {issue}48752[#48752])
+
+Authorization::
+* Allow kibana_system to create and invalidate API keys on behalf of other users {pull}53824[#53824] (issue: {issue}48716[#48716])
+* Add "grant_api_key" cluster privilege {pull}53527[#53527] (issues: {issue}48716[#48716], {issue}52886[#52886])
+* Giving kibana user privileges to create custom link index {pull}53221[#53221] (issue: {issue}59305[#59305])
+* [APM] Allow kibana to collect APM telemetry in background task {pull}52917[#52917] (issue: {issue}50757[#50757])
```

Comment:
nit: I try to remove the `[<category]` from the descriptions. But probably not a requirement. (here and a few places below)

Reply:
I've generally removed them, except for a few cases where I thought it might add some clarification. But here it's obviously redundant, so I've fixed it. Thanks.